SensusService/Probes/User/Scripts/ScriptRunner.cs

```diff
@@ -394,6 +408,23 @@ private void StartRandomTriggerCallbacksAsync()
             }).Start();
         }
 
+        private Task runScript(string callbackId)
```

Comment:
Nope, I just factored the inline lambda expression out. In your code it only appears once but I use it twice, so I thought I'd get rid of the verbatim repetition. Should I rename it?